libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -270,6 +270,10 @@ public async Task ProcessAsync(HttpRequest httpRequest, HttpResponse httpRespons
         /// <summary>
         /// Factory method to create the <see cref="FacebookMessage"/> instance of the <see cref="Activity"/> to be sent to Facebook.
         /// </summary>
+        /// <remarks>
+        /// This lets an override add a Facebook-supported message tag to an outgoing message.
```

Comment:
The PR is merged but just for future reference, this is just "one" possible use case and I wouldn't have added this comment. The dev can do whatever she think is necessary here.